styles/style.css

```diff
@@ -1,45 +1,42 @@
 body {
-    /* feel free to update these css variables for a quick theme */
+    
 
-    --color1: white;
+    --color1: rgb(255, 255, 255);
     --color2: white;
     --color3: white;
     --color4: white;
     --color5: white;
 
-    /* https://coolors.co/generate and click export -> scss to get the colors to copy and paste below*/
-
-    /* --color1: hsla(349, 69%, 76%, 1);
-    --color2: hsla(207, 67%, 44%, 1);
-    --color3: hsla(200, 61%, 56%, 1);
-    --color4: hsla(187, 41%, 94%, 1);
-    --color5: hsla(170, 28%, 95%, 1); */
-
-    /* --color1: hsla(45, 86%, 83%, 1);
-    --color2: hsla(135, 65%, 98%, 1);
-    --color3: hsla(315, 37%, 86%, 1);
-    --color4: hsla(45, 73%, 98%, 1);
-    --color5: hsla(224, 40%, 79%, 1); */
-
-    --header-height: 80px;
+    --header-height: 50px;
     --shadow: 0px 10px 20px -12px rgba(0, 0, 0, 0.45);
     --gutter: 30px;
     --footer-height: 30px;
 
-    font-family: 'Titillium Web', sans-serif;
-    background-color: var(--color2);
+    background-color: var(blue);
 }
 
 main {
-    display: flex;
-    flex-direction: column;
+    display: inside;
+    flex-direction: row;
```

Comment:
`flex-direction` and the other `flex-**` rules will only apply when the parent container is using `display: flex` for the CSS structure.